### ISSUE TITLE
build failed: Retrieving 'expat-2.2.6' [ERROR]    expat: download failed

### ISSUE THREAD

**Issue by Freddie322** [crosstool-ng/crosstool-ng#1502]
Hello
I try to build a compiler for the target : armv7-rpi2-linux-gnueabihf
Executed on Linux ubuntu 5.4.0-1015-raspi #15-Ubuntu SMP Fri Jul 10 05:34:24 UTC 2020 aarch64 aarch64 aarch64 GNU/Linux
And it fails on expat-2.2.6 tarball retrieving.
it seems that the expat actual version is 2.3.0 and not 2.2.6
this is the log
[build.log](https://github.com/crosstool-ng/crosstool-ng/files/6410770/build.log)
Regards

**Comment by cpackham** [comment]
See #1479

**Comment by Freddie322** [comment]
In that thread you talk about setting CT_EXPERMENTAL... I am new in this environment and do not know  where to set it...
Finally I did change 2.2.6 version to 2.3.0 in config/versions/expat.in file and save the config by menuconfig and tryed to build.
Later, I had another issue :failing of the test of python3.  I had to add the python3-dev package to my system.
And then it runs like a charm ( on my ubuntu 20 on an raspberry 4 it take few hours...)
